recipes-dotnet/coreclr/coreclr/toolchain.patch

```diff
@@ -20,16 +20,16 @@ index 84ae2a5..17fdf15 100644
  
  ## Specify the toolchain
 -set(TOOLCHAIN "arm-linux-gnueabihf")
-+set(TOOLCHAIN "arm-target-linux-gnueabi")
++set(TOOLCHAIN "arm-poky-linux-gnueabi")
```

Comment:
Hmm, I had convinced myself that the compiler name is actually `arm-target-linux-gnueabi`, but it looks like the `target` and `poky` parts are the name of the distro (my distro is called `target` as in the company name of my employer).

That means this simple patch approach doesn't suffice :-/